Remove Personalisations from Dashboard

theoisadoor opened the pull request:
Personalisations were added to the dashboard which is not fit for use by other users. Since this dashboard is [referenced directly by the speedtest-tracker Wiki](https://docs.speedtest-tracker.dev/settings/data-platforms/influxdb2#grafana-dashboard), this repo should be as standard as possible.

- Variables from buckets were updated from `speedtest-tracker-new,speedtest-tracker-cloud` > `speedtest-tracker` since [this is the expected variable for new/basic installations](https://docs.speedtest-tracker.dev/settings/data-platforms/influxdb2#settings)
- Links to speedtest-tracker application updated from `https://192.168.0.199:9443` > `https://192.168.0.0:8443` since the [default port is 8443 across all installation types](https://docs.speedtest-tracker.dev/getting-started/installation), removed last octet since this is also bespoke.
- Removed personalisation from link `Speedtest Tracker - DaRKNaS` > `Speedtest Tracker`
- Updated README.md

theoisadoor committed:
Update speedtest-tracker-v2.json

Removed personalisation from link (removed DaRKNaS)
Changed IP from 192.168.0.199 > 192.168.0.0 for obscurity
Change port from 9443 (personalised) to 8443 (default exposed port for Speedtest-Tracker v2)
Replaced `speedtest-tracker-new,speedtest-tracker-cloud` with `speedtest-tracker` which is the expected default bucket

theoisadoor committed:
Update README.md

Updated README.md with changes.

masterwishx commented:
Thanks for the pr, seems DaRKNaS was added by mistake.
Also my unraid server is down for some time, motherboard is broken, waiting for new parts.

theoisadoor committed:
Update README.md

Updated README.md
- spelling issues
- improved installation instructions
- updated screenshots

theoisadoor committed:
Update README.md
